doc/DEPLOYMENT.md

```diff
@@ -30,6 +30,8 @@ This section covers the process of setting up an EC2 instance on AWS that will b
 
 ### Create EC2 instance for deployment
 
+**TODO: update this section; use launch template**
```

Comment:
Is there a ticket for this yet?  Might be good next sprint.

Reply:
Now there is :)  https://jira.stsci.edu/browse/JUSI-641